lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -118,6 +119,8 @@ private void init(@Nullable AttributeSet attrs) {
       addColorFilter(new SimpleColorFilter(ta.getColor(
           R.styleable.LottieAnimationView_lottie_colorFilter, Color.TRANSPARENT)));
     }
+
+    scale = ta.getFloat(R.styleable.LottieAnimationView_lottie_scale, Float.MAX_VALUE);
```

Comment:
`composition` might be not be loaded, so it can't calculate the right scale with screen size.